apps/explorer/src/components/node-map/index.tsx

```diff
@@ -43,7 +43,7 @@ function NodeStat({ title, children }: { title: string; children: ReactNode }) {
 }
 
 interface Props {
-    minHeight: number;
+    minHeight: string | number;
```

Comment:
I see you are passing in string percentage. Will it work if it is just a flat string number?

Reply:
nah - it'll render as `100px` unfortunately